specs/latest/1.0/index.html

```diff
@@ -3565,10 +3565,7 @@ <h2 class="no-toc">Table of contents</h2>
 <h3><a name="READS_OUTSIDE_FRAMEBUFFER">Reading Pixels Outside the Framebuffer</a></h3>
 
 <p>
-In the WebGL API, functions which read the framebuffer
-(<code>copyTexImage2D</code>, <code>copyTexSubImage2D</code>, and <code>readPixels</code>) are
-defined to generate the RGBA value (0, 0, 0, 0) for any pixel which is outside of the bound
-framebuffer.
+In the WebGL API, three functions read the framebuffer: <code>copyTexImage2D</code>, <code>copyTexSubImage2D</code>, and <code>readPixels</code>. <code>copyTexImage2D</code> is defined to generate the RGBA value (0, 0, 0, 0) for any pixel outside of the bound framebuffer. <code>copyTexSubImage2D</code> and <code>readPixels</code> are defined not to touch the corresponding destination range for any pixel outside the bound framebuffer.
```

Comment:
CopyTexImage should have the same OOB behavior.

The guarantee should be that CopyTexImage2D functions as TexImage2D(null)+CopyTexSubImage2D. That is, if the texture data is previously undefined, we define it when we write to a subset of the pixels.